README.md

```diff
@@ -156,6 +156,19 @@ On certain pages an API endpoint is called to receive experiment analysis data f
    GOOGLE_APPLICATION_CREDENTIALS=/app/google-credentials.json
    ```
 
+### Google Credentials for Media Storage
+
+We support user uploads of media (e.g. screenshots) for some features.
+
+In local development, the default is to store these files `/app/media` using Django's `FileSystemStorage` class and the `MEDIA_ROOT` and `MEDIA_URL` settings.
+
+In production, GCP credentials and a bucket are required. These are configured with the `UPLOADS_GS_CREDENTIALS` and `UPLOADS_GS_BUCKET_NAME` settings respectively. For example:
+
+```
+UPLOADS_GS_CREDENTIALS=/app/nimbus-experimenter-media-dev-54693c3a8c69.json
+UPLOADS_GS_BUCKET_NAME=nimbus-experimenter-media-dev-uploads
+```
+
```

Comment:
One more quick tweak to document the new settings. Will probably merge after CI is green